library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionOverviewFragment.kt

```diff
@@ -80,6 +89,13 @@ internal class TransactionOverviewFragment : Fragment() {
                     status.text = transaction?.status?.toString()
                     response.text = transaction?.responseSummaryText
                     ssl.setText(if (transaction?.isSsl == true) R.string.chucker_yes else R.string.chucker_no)
```

Comment:
Can you move this also inside the `if` branch?

Reply:
👌